drivers/wifi/wfx200/Kconfig.wfx200

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) 2021 grandcentrix GmbH
+# 
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig WIFI_WFX200
+	bool "SiLabs WFX200 support"
+    select HAS_SILABS_WFX_FMAC
+	depends on NET_L2_WIFI_MGMT
+	depends on WIFI_OFFLOAD
+    depends on NET_L2_ETHERNET
+    help
+      SiLabs WFX200 WiFi Module with SPI
+      or SDIO Interface
+
+if WIFI_WFX200
+
+choice WIFI_WFX200_BUS
+    bool "Select BUS interface"
+    default WIFI_WFX200_BUS_SPI
+
+config WIFI_WFX200_BUS_SPI
+    bool "SPI Bus interface"
+    select SPI
+
+#config WIFI_WFX200_BUS_SDIO
+#    bool "SDIO Bus interface"
+#    help
+#      Not currently supported.
+
+endchoice # WIFI_WFX200_BUS
```

Comment:
I'm confused about this. Can't this come from devicetree? See drivers/sensor/bme280 for an example driver that can support nodes on different buses and just does the right thing.

Reply:
Yeah, i just started somewhere ...
But if i'm not totally mistaken, there is still no generalized SDIO driver model in zephyr, right? I saw some SD-Card drivers to use SDIO peripherals from manufacturer SDKs. Wouldn't it be feasible to just remove all the SDIO parts from this driver and implement this driver SPI-only? If zephyr gets an SDIO driver model sometime in the future, we can still add the SDIO bus option later.